<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="Your description goes here" />
	<meta name="keywords" content="your,keywords,goes,here" />
	<meta name="author" content="Your Name" />
	
	<link rel="stylesheet" type="text/css" href="test.css" title="Sun Lab" media="all" />
	<title>SUN Lab</title>
</head>

<body class="wider ">
<div id="layout">

	<div id="header" class="row">
		<div class="col c4">
			<h1><a class="red" href="index.html">SUN Lab</a></h1>
			
		</div>
		
		<div class="col c2" >
			<ul class="menu">
				<li><a class="red" href="research.html">RESEARCH</a></li>
				
			</ul>
		</div>
		<div class="col c2">
			<ul class="menu">
				<li ><a class="red" href="people.html">PEOPLE</a></li>
				
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="publications.html">PUBLICATION</a></li>
				
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="software.html">SOFTWARE</a></li>
				
			</ul>
		</div>

		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="teaching.html">TEACHING</a></li>
				
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="Blog Posts.html">BLOG</a></li>
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="contact.html">CONTACT</a></li>
			</ul>
		</div>
	</div>
	<img src="img/2.jpg" alt="西交" width="100%" height="250px"/>
	
    
	
	<div class="row">
		<div class="col c13">
			<div id="headerimage" class="col c12">
			<p><a href="index.html"><img src="img/quan.jpg" alt="The Grid template" height="120px" width="120px"/></a></p>
		    </div>
			<p><b>Shiquan Sun, PhD</b></p>
			
			<p>Professor of Biostatistics at Xi'an Jiaotong University</p>
			<p><img src="img/定位.png" alt="The Grid template" height="18px" width="18px"/> Shaanxi, China</p>
			<p><img src="img/twitter.png" alt="The Grid template" height="18px" width="18x"/><a href="https://twitter.com/SunShiquan"  > Twitter</a> </p>
			<p><img src="img/github.png" alt="The Grid template" height="18px" width="18px"/><a href="https://github.com/sqsun"  > Github</a> </p>
			<p><img src="img/google.png" alt="The Grid template" height="18px" width="18px"/><a href="https://scholar.google.com/citations?user=fSihIqYAAAAJ&hl=zh-CN&oi=sra"  > Google Scholar</a> </p>
			<p><img src="img/orcid.png" alt="The Grid template" height="18px" width="18px"/><a href="https://orcid.org/0000-0002-9150-6992"  > Orcid</a> </p>
		</div>
		
		<div class="row1">
		<div class="col c3">
			<br />
			<h2 style="font-weight:700 ;">Research</h2>
			<br />
			<h3 style="font-weight:700 ;">Methods for Single Cell Analysis</h3>
			
			<p> Single-cell sequencing (scSeq) technology gives unprecedented insights into complex tissues (e.g., brain) at the level of individual cells, 
			    allowing for more precise research on cell heterogeneity. In contrast to traditional sequencing technologies (e.g., bulk sequencing), 
			    scSeq enables analysis of hundreds of thousands individual cells simultaneously. Such transformative new methods would be able to profile genomic, epigenomic, proteomic 
			    and metabolic information in an individual cell. This of course provides unique opportunities, alongside computational challenges, 
			    for statistical and computational methods that can jointly learn across multiple types of data. For example, how to define the cell states, 
			    how to identify the rare cell subpopulations, how to deal with batch effects, and how can we select the proper method for downstream analysis, etc.
			</p>
			
			<hr />
			
			
			<h3 style="font-weight:700 ;">Methods for Spatially Resolved Transcriptomics Analysis</h3>
			
			<p> Recent advancements of sequencing technologies in molecular biology have enabled us to generate gene expression data with spatial location information. 
			    Identifying genes that display certain spatial expression (SE) patterns can further our understanding of the tissue organization 
			    and the spatial transcriptome landscape. It has very important applications in the identification of tissue function, tracking of developmental process, 
			    pathological detection of cancer tissue slices and combined application of molecular detection.
				
			    We present such a statistical method, Spatial PAttern Recognition via Kernels (SPARK), as an effective statistical tool to identify genes 
			    with spatial expression patterns in spatially resolved transcriptomic studies. 
			    A particular feature of SPARK is its ability to produce calibrated p-values and powerful results for spatial analysis.
			    
			</p>
			
		</div>

		
	</div>
	</div>
	

	

	<div id="footer" class="row">
		<div class="col c14">
			<p><b>Sitemap</b><p />
			<p><b>Follow:</b>&nbsp;&nbsp;&nbsp; <img src="img/github.png" alt="The Grid template" height="15px" width="15px"/><a href="https://github.com/sqsun"  >   <b>GITHUB</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</a>
				<img src="img/feed3.png" alt="The Grid template" height="15px" width="15px"/><a href="https://sqsun.github.io/feed.xml"  >  <b>FEED</b> </a>
			</p>
			<p>&copy; 2020 Dr. Shiquan Sun. Powered by <a href="https://jekyllrb.com/"> Jekyll </a> & <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/"> Minimal Mistakes.</a> </p>
		</div>
	</div>
</div>
</body>
</html>
