<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.5.0/jquery.min.js"
	type="text/javascript"></script>
<script type="text/javascript">
$(document).ready(function(){
	$("#expanderHead").click(function(){
		$("#expanderContent").slideToggle();
		if ($("#expanderSign").text() == "+"){
			$("#expanderSign").html("−")
		}
		else {
			$("#expanderSign").text("+")
		}
	});
});
function toggleSlides(){
		//$('.slider').addClass('sliderHide'); if no JS enabled, slides are open, they close after page is loaded
		$('.toggler').click(function(e){
			var id=$(this).attr('id');
			var widgetId=id.substring(id.indexOf('-')+1,id.length);
			$('#'+widgetId).slideToggle();
			$(this).toggleClass('sliderExpanded');
			$('.closeSlider').click(function(){
					$(this).parent().hide('slow');
					var relatedToggler='toggler-'+$(this).parent().attr('id');
					$('#'+relatedToggler).removeClass('sliderExpanded');
			});
		});
	};
	$(function(){
		toggleSlides();
	});
</script>
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="Your description goes here" />
	<meta name="keywords" content="your,keywords,goes,here" />
	<meta name="author" content="Your Name" />
	<link href='http://fonts.googleapis.com/css?family=Dosis' rel='stylesheet' type='text/css' />
	<link rel="stylesheet" type="text/css" href="test.css" title="Sun Lab" media="all" />
	<script type="text/javascript" src="js/jquery-1.7.1.min.js"></script>
	<script type="text/javascript" src="js/jquery.nivo.slider.js"></script>
	<title>SUN Lab</title>
</head>

<body class="wider ">
<div id="layout">

	<div id="header" class="row">
		<div class="col c4">
			<h1><a class="red" href="index.html">SUN Lab</a></h1>
			
		</div>
		
		<div class="col c2" >
			<ul class="menu">
				<li><a class="red" href="research.html">RESEARCH</a></li>
				
			</ul>
		</div>
		<div class="col c2">
			<ul class="menu">
				<li ><a class="red" href="people.html">PEOPLE</a></li>
				
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="publications.html">PUBLICATION</a></li>
				
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="software.html">SOFTWARE</a></li>
				
			</ul>
		</div>

		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="teaching.html">TEACHING</a></li>
				
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="Blog Posts.html">BLOG</a></li>
			</ul>
		</div>
		
		<div class="col c2">
			<ul class="menu">
				<li><a class="red" href="contact.html">CONTACT</a></li>
			</ul>
		</div>
	</div>
	<img src="img/2.jpg" alt="西交" width="100%" height="250px"/>

    
	
	<div class="row">
		<div class="col c13">
			<div id="headerimage" class="col c12">
			<p><a href="index.html"><img src="img/quan.jpg" alt="The Grid template" height="120px" width="120px"/></a></p>
		    </div>
			<p style="font-size: 16px;"><b>Shiquan Sun, PhD</b></p>
			<p>Professor of Biostatistics at Xi'an Jiaotong University</p>
			<p><img src="img/定位.png" alt="The Grid template" height="18px" width="18px"/> Shaanxi, China</p>
			<p><img src="img/twitter.png" alt="The Grid template" height="18px" width="18x"/><a href="https://twitter.com/SunShiquan"  > Twitter</a> </p>
			<p><img src="img/github.png" alt="The Grid template" height="18px" width="18px"/><a href="https://github.com/sqsun"  > Github</a> </p>
			<p><img src="img/google.png" alt="The Grid template" height="18px" width="18px"/><a href="https://scholar.google.com/citations?user=fSihIqYAAAAJ&hl=zh-CN&oi=sra"  > Google Scholar</a> </p>
			<p><img src="img/orcid.png" alt="The Grid template" height="18px" width="18px"/><a href="https://orcid.org/0000-0002-9150-6992"  > Orcid</a> </p>
		</div>
		
		<div class="row1">
		<div class="col c3">
			
			<br />
			<h2 style="font-weight:700 ;">Software</h2>
			<br />
			
		<details class="menu" >
			<summary><h4 style="font-weight:700 ;">SPARK -- Spatial PAttern Recognition via Kernels</h4></summary>
			<ul>
			<li>
			<div class="slider" id="slideEleven" >

                <p> SPARK is a method for detecting genes with spatial expression patterns in spatially resolved transcriptomic studies. SPARK directly models count data generated from various spatial resolved transcriptomic techniques through generalized spatial linear models. With a new efficient penalized quasi-likelihood based algorithm, SPARK is scalable to data sets with tens of thousands of genes measured on tens of thousands of samples. Importantly, SPARK relies on newly developed statistical formulas for hypothesis testing, producing well-calibrated p-values and yielding high statistical power. The software is distributed under the <a href="http://www.gnu.org/copyleft/gpl.html"> GNU General Public License</a>.  </p>

                <p><b>Download</b></p>
                <ul>
                <p> The package is currently available at <a href="https://xzhoulab.github.io/SPARK/">github</a>. </p>
                </ul>

                <p><b>Citations</b></p>
                <ul>
                <p>Shiquan Sun*, Jiaqiang Zhu* and Xiang Zhou (2020). Statistical analysis of spatial expression pattern for spatially resolved transcriptomic studies. <i>Nature Methods</i>. 17: 193-200.
                </p>
                <p> All analysis scripts used in the paper is available at <a href="https://github.com/xzhoulab/SPARK-Analysis">github</a>. </p>
                </ul>

                <p><b>Contact <a href="mailto:shiquasun@gmail.com"> me </a> or <a href="mailto:jiaqiang@umich.edu"> Jiaqiang Zhu </a> with any questions, comments, or bugs reports.</b></p>
            </div>
           </ul>
			</details>
			<br />
		
			<details class="menu" >
			<summary><h4 style="font-weight:700 ;">SpatialMap -- Spatial Mapping unmeasured genes with scRNA-seq data</h4></summary>
			<ul>
			<li>
			<div class="slider" id="slideEleven" >

                <p>  SpatialMap is a computational method, which primarily facilitates spatial mapping of unmeasured gene profiles in spatial transcriptomic data via integrating with scRNA-seq data from the same tissue. SpatialMap directly models the count nature of spatial gene expression data through generalized linear spatial models, which accounts for the spatial correlation among spatial locations using conditional autoregressive (CAR) prior. The software is distributed under the <a href="http://www.gnu.org/copyleft/gpl.html"> GNU General Public License</a>.  </p>

                <p><b>Download</b></p>
                <ul>
                <p> The package is currently available at <a href="http://github.com/JingNing329/SpatialMap">github</a>. </p>
                </ul>

                <p><b>Citations</b></p>
                <ul>
                <p>Dalong Gao*, Jin Ning*, Gang Liu, Shiquan Sun# and Xiaoqian Dang# (2022).  SpatialMap: Spatial mapping of unmeasured gene expression profiles in spatial transcriptomic data using generalized linear spatial models. <i> Front. Genet.</i> 13:893522. doi: 10.3389/fgene.2022.893522.
                </p>
                <p> All analysis scripts used in the paper is available at <a href="http://github.com/JingNing329/SpatialMap">github</a>. </p>
                </ul>

                <p><b>Contact <a href="mailto:newlife1012@stu.xjtu.edu.cn"> Jin Ning </a> with any questions, comments, or bugs reports.</b></p>
            </div>
	  </ul>
			</details>
			<br />
		
			<details class="menu" >
			<summary><h4 style="font-weight:700 ;">TDEseq -- Temporal Differential Expression analysis of time course scRNA-seq data</h4></summary>
			<ul>
			<li>
			<div class="slider" id="slideEleven" >

                <p>  TDEseq is a computational method for detecting genes with temporal dynamic expression patterns in time course scRNA-seq transcriptomic studies. TDEseq models the relationship between log normalized data and the corresponding time points through constrained additive mixed model and can detect the DE genes as well as its temporal dynamic pattern simultaneously. The software is distributed under the <a href="http://www.gnu.org/copyleft/gpl.html"> GNU General Public License</a>.  </p>
                <p><b>Download</b></p>
                <ul>
                <p> The package is currently available at <a href="https://github.com/fanyue322/TDEseq">github</a>. </p>
                </ul>

                <p><b>Citations</b></p>
                <ul>
                <p>Yue Fan, Lei Li, Dayun Feng, and Shiquan Sun# (2022). TDEseq: detecting temporal gene expression changes in time course single-cell RNA sequencing studies. <i> xxx.</i> xxx.
                </p>
                <p> All analysis scripts used in the paper is available at <a href="https://github.com/fanyue322/TDEseq">github</a>. </p>
                </ul>

                <p><b>Contact <a href="mailto:xafanyue@xjtu.edu.cn"> Yue Fan </a> with any questions, comments, or bugs reports.</b></p>
            </div>
           </ul>
			</details>
			<br />	
			
            <details class="menu" >
			<summary><h4 style="font-weight:700 ;">iDEA -- integrative Differential expression and gene set Enrichment Analysis</h4></summary>
			<ul>
			<li>
				<div class="slider" id="slideTwelve" >

                <p> iDEA is a method for performing joint differential expression (DE)and gene set enrichment (GSE) analysis. iDEA builds upon a hierarchical Bayesian model for joint modeling of DE and GSE analyses. It uses only summary statistics as input, allowing for effective data modeling through complementing and pairing with various existing DE methods. It relies on an efficient expectation-maximization algorithm with internal Markov Chain Monte Carlo steps for scalable inference. By integrating DE and GSE analyses, iDEA can improve the power and consistency of DE analysis and the accuracy of GSE analysis over common existing approaches. The software is distributed under the <a href="http://www.gnu.org/copyleft/gpl.html"> GNU General Public License</a>.  </p>
                <p><b>Download</b></p>
                <ul>
                <p> The package is currently available at <a href="https://xzhoulab.github.io/iDEA/">github</a>. </p>
                </ul>

                <p><b>Citations</b></p>
                <ul>
                <p> Ying Ma*, Shiquan Sun*, Xuequn Shang, Evan T. Keller, Mengjie Chen and Xiang Zhou. (2020) Integrative differential expression and gene set enrichment analysis using summary statistics for scRNA-seq studies.<i>Nature Communications</i>. 11: 1585.</p>
                <p> All analysis scripts used in the paper is available at <a href="https://github.com/xzhoulab/iDEA-Analysis">github</a>. </p>
                </ul>

                <p><b>Contact <a href="mailto:shiquasun@gmail.com"> me </a> or <a href="mailto:yingma@umich.edu"> Ying Ma </a> with any questions, comments, or bugs reports.</b></p>
            </div>
			</a></li>
			</ul>
			</details>
			
            <br />
            
            <details class="menu" >
			<summary><h4 style="font-weight:700 ;">MACAU -- Mixed model Association for Count data via data AUgmentation</h4></summary>
			<ul>
			<li>
			<div class="slider" id="slideTwo" >

                <p>MACAU is the software implementing the Mixed model Association for Count data via data AUgmentation algorithm. It fits a binomial mixed model to perform differential methylation analysis for bisulfite sequencing studies. It fits a Poisson mixed
                model to perform differential expression analysis for RNA sequencing studies. It is computationally efficient for large scale sequencing studies and uses freely available open-source numerical libraries. It is distributed under the <a href="http://www.gnu.org/copyleft/gpl.html"> GNU General Public License</a>.  </p>
                <p><b>C/C++ Version Download</b></p>
                <ul>
                <li> <a href="software/macau/macau" onClick="_gaq.push(['_trackEvent','Software', 'Downloaded','MACAU_executable']);">Executable</a>, Version 1.40, for x86 64-bit Linux, compiled on 08/09/2019 </li>
                <li> <a href="software/macau/macau-v1.40.rar" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_source']);">Source Code</a>,Version 1.40, last modified on 08/09/2019 </li>

                <li> <a href="software/macau-example.tar.gz" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_example']);">Example Data Sets</a>: a bisulfite sequencing data with 438,865 methylation sites for 50 baboons; and an RNA squencing data with 12,018 genes for 63 baboons. </li>

                <li> <a href="software/MACAUmanual.pdf" onClick="_gaq.push(['_trackEvent', 'Software', 'Downloaded','MACAU_manual']);">User Manual</a>, last edited on 03/15/2017 </li>

                <li> <a href="software/bb.r" onClick="_gaq.push(['_trackEvent','Software', 'Downloaded','BetaBinomial']);"> R script</a>  to fit a beta-binomial model</li>
                </ul>
                <p><b>R Version Download</b></p>
                <ul>
                <li> <a href="software/MACAU-R-Manual.pdf" onClick="_gaq.push(['_trackEvent', 'Software', 'Downloaded','MACAU_manual']);">User Manual</a>, last edited on 04/09/2017 </li>
                <li> <a href="software/MACAU-R-ReferenceManual.pdf" onClick="_gaq.push(['_trackEvent', 'Software', 'Downloaded','MACAU_manual']);">R Reference Manual</a>, last edited on 04/08/2017 </li>
                <li> <a href="software/macau-2.00.tar.gz" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_example']);">Source Code</a>,Version 2.00, last modified on 04/09/2017 </li>
                </ul>

                <p><b>Update Log</b></p>
                <ul>
<!--<li>04/09/2017: version 2.00. R package with a new algorithm and parallele computing
    capacity. </li>-->
               <li>08/09/2019: version 1.40. Shiquan extended the model to unrelated indivdiuals. PMM can now fit data without providing a kinship matrix. </li>
               <li>05/27/2019: <a href="software/macau-v1.30.rar" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_example']);">version 1.30</a>. Shiquan fixed a bug on reading methylation count data. </li>
               <li>03/22/2018: <a href="software/macau-1.20.tar.gz" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_example']);">version 1.20</a>. Shiquan fixed a bug, where the heritability estimate always equals 0.5 when the sample size is larger than 200. </li>
               <li>03/15/2017: <a href="software/macau-1.10.tar.gz" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_example']);">version 1.10</a>. implements the Poisson mixed model for differentia expression analysis in RNA sequencing studies. </li>
               <li>12/09/2015: <a href="software/macau-example.tar.gz" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','MACAU_example']);">version 1.00</a>. implements the Binomial mixed model for differentia methylation analysis in bisulfite sequencing studies. </li>
               <li>06/05/2015: version 1.00alpha, initial release. </li>
               </ul>


               <p><b>Citations</b></p>
               <ul>
               <li>Binomial mixed models for bisulfite sequencing analysis</li>&nbsp; &nbsp; &nbsp; Amanda J. Lea, Jenny Tung and Xiang Zhou (2015). A flexible, effcient binomial mixed model for identifying differential DNA methylation in bisulfite sequencing data. PLoS Genetics. 11: e1005650.

               <li>Poisson mixed models for RNA sequencing analysis</li> &nbsp; &nbsp; &nbsp;Shiquan Sun, Michelle Hood, Laura Scott, Qinke Peng, Sayan Mukherjee, Jenny Tung, and Xiang Zhou. (2017) Differential expression analysis for RNAseq using Poisson mixed models. <i>Nucleic Acids Research</i> 45: e106.</p>
               </ul>

               <p><b>Contact <a href="mailto:shiquasun@gmail.com">me</a> with any questions, comments, or bugs reports.</b></p>

            </div>
            </ul>
			</details>
			
			<br />
			
			<details class="menu" >
			
			<summary><h4 style="font-weight:700 ;">PQLseq -- Penalized QuasiLikelihood for sequencing count data</h4></summary>
			<ul>
			<li>
			<div class="slider" id="slideSix" >
               <p> PQLseq is a method that fits generalized linear mixed models for analyzing RNA sequencing and bisulfite sequencing data. It estimates gene expression or methylation heritability for count data. It performs differential expression analysis in the presence of individual relatedness or population stratificaiton. The software is distributed under the <a href="http://www.gnu.org/copyleft/gpl.html"> GNU General Public License</a>.  </p>
               <p> The package is available in <a href="https://cran.r-project.org/web/packages/PQLseq/index.html">CRAN</a>
               <p><b>Download</b></p>
               <ul>
               <li> <a href="software/PQLseqManual.pdf" onClick="_gaq.push(['_trackEvent', 'Software', 'Downloaded','PQLseq_manual']);">User Manual</a>, last edited on 03/20/2018 </li>
               <li> <a href="https://cran.r-project.org/web/packages/PQLseq/index.html" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','PQLseq_src']);">CRAN R Package</a>,Version 1.0, last modified on 06/10/2018 </li>
               </ul>

               <p><b>Update Log</b></p>
               <ul>
               <li>03/20/2018: version 0.10, initial release; now available in <a href="https://cran.r-project.org/web/packages/PQLseq/index.html" onClick="_gaq.push(['_trackEvent', 'Software','Downloaded','PQLseq_src']);">CRAN</a>. </li>
               </ul>
               <p><b>Citations</b></p>
               <ul>
               <p>Shiquan Sun*, Jiaqiang Zhu*, Sahar Mozaffari, Carole Ober, Mengjie Chen, and Xiang Zhou. (2019) Heritability estimation and differential analysis of count data with generalized linear mixed models in genomic sequencing studies. <i>Bioinformatics</i> 35: 487-496.</p>

               </ul>

               <p><b>Contact <a href="mailto:shiquasun@gmail.com"> me </a> with any questions, comments, or bugs reports.</b></p>
            </div>
			</ul>
			</details>
			
		</div>

		
	</div>
	</div>
	

	

	<div id="footer" class="row">
		<div class="col c14">
			<p><b>Sitemap</b><p />
			<p><b>Follow:</b>&nbsp;&nbsp;&nbsp; <img src="img/github.png" alt="The Grid template" height="15px" width="15px"/><a href="https://github.com/sqsun"  >   <b>GITHUB</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</a>
				<img src="img/feed3.png" alt="The Grid template" height="15px" width="15px"/><a href="https://sqsun.github.io/feed.xml"  >  <b>FEED</b> </a>
			</p>
			<p>&copy; 2020 Dr. Shiquan Sun. Powered by <a href="https://jekyllrb.com/"> Jekyll </a> & <a href="https://github.com/academicpages/academicpages.github.io">AcademicPages</a>, a fork of <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/"> Minimal Mistakes.</a> </p>
		</div>
	</div>
</div>
</body>
</html>
